corgi

```diff
@@ -189,6 +189,14 @@ elif [[ "$COMMAND" == "create-approved-books" ]]; then
     token="${1:-fake-token}"
     docker compose -f "$dev_stack" exec -T backend \
         python - "$token" < "$here/scripts/add-approved-books.py"
+elif [[ "$COMMAND" == "lint" ]]; then
+    docker compose -f "$dev_stack" run --rm backend ruff check
+    docker compose -f "$dev_stack" run --rm backend ruff format --check
+    docker compose -f "$dev_stack" run --rm frontend npm run lint
+elif [[ "$COMMAND" == "format" ]]; then
```

Comment:
Nice, `lint` for checking and `format` for changing (as opposed to something like `lint` and `lint:fix`)